src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -248,7 +248,7 @@ internal static class TrustedSignersCommand
 
             try
             {
-                ISettings settings = ProcessConfigFile(configFile.Value());
+                ISettings settings = XPlatUtility.ProcessConfigFile(configFile.Value());
```

Comment:
Does this command suffer form the same problem as the verify command?

Reply:
No, it doesn't. I just moved `ProcessConfigFile` method to common utility path.

Reply:
So that'd imply that the process config file part is the fix?

Reply:
My qualm with is that it's not obvious what it does. 

What if the config value is not set? 
What's default settings? 
User config? wrt to some directory. 

Obviously most of this code is not written by you, so I'm just wondering if we can do better. 

Do my concerns make sense?

Reply:
Here `default settings` is normal setting which we use for restore/pack, it starts with machine wide settings then overridden with values closer to current working folder.
https://github.com/NuGet/NuGet.Client/blob/f24bad0668193ce21a1db8cabd1ce95ba509c7f0/src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/XPlatUtility.cs#L47-L53

Other option than default settings is we can pass relative/absolute path nuget.config. 
Actually, this `ProcessConfigFile`  is my thing I added when I implemented `dotnet nuget trust`, idea is taken from NuGet.CommandLine, I just modified bit for my need.

https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs#L122-L146

Reply:
I see. That makes sense. That means the PR is pretty complete. 

I think we can name the `CreateDefaultSettigns` better. 
It raises a lot of questions about what default settings really is. 

Should we be passing in the current directory? Cause realistically, it is GetSettingsForCurrentWorkingDirectory. 

Alternatively that could be the name.

Reply:
Ok. Just renamed to `GetSettingsForCurrentWorkingDirectory`.